packages/flutter/lib/src/widgets/sliver.dart

```diff
@@ -195,6 +202,15 @@ abstract class SliverChildDelegate {
   }
 }
 
+/// A Key that is used in [SliverChildBuilderDelegate] and [SliverChildListDelegate]
+/// that provide the wrapping logic same as ValueKey<Key>.
+class _SaltedValueKey extends ValueKey<Key>{
+  /// Creates a key that delegates its [operator==] to the given key.
+  const _SaltedValueKey(Key key):
+        assert(key != null),
+        super(key);
+}
+
```

Comment:
you can probably omit all the documentation on this class. it's private anyway.

Reply:
removed